docs/concepts/storage.md

```diff
@@ -0,0 +1,89 @@
+---
+id: storage-solutions
+title: Decentralized Storage solutions
+sidebar_label: Decentralized Storage
+---
+
+In this article, you'll find a brief overview of different decentralized storage solutions.
+These solutions can be integrated into your distributed applications (dApps), allowing you to access large amounts of data in a decentralized manner.
+
+#### Solutions
+
+- [Arweave](#arweave)
+- [IPFS](#ipfs)
+- [Sia](#sia)
+
+## Arweave
+
+[Arweave](https://www.arweave.org/) is a new type of storage that backs data with sustainable and perpetual endowments
+(donations to a nonprofit organization). This allows users and developers to store data forever.
+Arweave acts as a collectively owned hard drive, and allows their users to preserve valuable information, apps, and history indefinitely.
+
+The Arweave protocol matches people who have hard drive space to spare with those individuals and organizations that need to store data or host content permanently. This is achieved in a decentralized network, and all data stored is backed by a [sustainable endowment](https://arwiki.wiki/#/en/storage-endowment) ensuring it is available in perpetuity.
+
+:::info
+To learn more about Arweave, check its [mining mechanism](https://arwiki.wiki/#/en/arweave-mining) and its [bandwidth-sharing system](https://arwiki.wiki/#/en/karma). 
+:::
+
+## IPFS
+
+The [InterPlanetary File System](https://ipfs.io/) (IPFS) is a protocol and peer-to-peer network for storing and sharing data in a distributed file system. IPFS uses content-addressing to uniquely identify each file in a global namespace connecting all computing devices.
+
+### Content identifier
+
+When you add a file to IPFS, your file is split into smaller chunks, cryptographically hashed, and given a unique fingerprint called a content identifier (CID). 
+
+> **Note:** the CID acts as an permanent record of a file as it exists at that point in time.
+
+### Look-up
+
+When a node looks up for a file, it ask the peer nodes for the content referenced by the file's CID. When a node views or downloads a file, it caches a copy and become another provider until the cache is cleared. 
+
+### Pinned content
+
+On the IPFS network, each node stores only content it is interested in.
+A node can pin content in order to keep it forever, or discard content it hasn't used to save space.
+
+### File versions
+
+When you add a new version of your file to IPFS, since the cryptographic hash is different, it will get a new CID.
+This means that any changes to a file don't overwrite the original, and common chunks across files can be reused in order to minimize storage costs.
+
+### Naming system
+
+IPFS offers a decentralized naming system so you don't need to remember a long string of CIDs.
+IPFS can find the latest version of your file using the IPNS decentralized naming system, and you can use DNSLink to map CIDs to human-readable DNS names.
+
+### IPFS providers
```

Comment:
Is it possible to make all these headers one size down? It's kinda hard to notice the difference between the main sections (SIA, IPFS, ARWEAVE), and their subheadings as shown here: 
![image](https://user-images.githubusercontent.com/57506486/137491066-07272579-832a-4848-973f-f377e466f075.png)